src/amo/reducers/guide.js

```diff
@@ -0,0 +1,72 @@
+/* @flow */
+import invariant from 'invariant';
+
+import { createPlatformFiles } from 'core/reducers/addons';
+import type { AddonType } from 'core/types/addons';
+
+export const FETCH_GUIDE_ADDONS: 'FETCH_GUIDE_ADDONS' = 'FETCH_GUIDE_ADDONS';
+export const LOAD_GUIDE_ADDONS: 'LOAD_GUIDE_ADDONS' = 'LOAD_GUIDE_ADDONS';
+
+export type GuideType = {|
+  addons: Array<AddonType> | void,
+|};
+export type FetchGuideParams = {|
+  errorHandlerId: string,
+  guid: string,
+|};
+export type FetchGuideAction = {|
+  type: typeof FETCH_GUIDE_ADDONS,
+  payload: FetchGuideParams,
+|};
+export type LoadGuideAction = {|
+  type: typeof LOAD_GUIDE_ADDONS,
+  payload: GuideType,
+|};
+export const initialState: GuideType = {
+  addons: undefined,
+};
+export const loadGuideAddons = ({ addons }: GuideType): LoadGuideAction => {
+  invariant(addons, 'addons is required');
+
+  const mappedResults =
+    addons &&
+    addons.map((addon) => {
+      return {
+        ...addon,
+        platformFiles: createPlatformFiles(addon.current_version),
```

Comment:
Why exactly do you need this?

Actually, I am not sure add-ons should be stored in this reducer. We have a reducer (`addons`) for the add-ons. I'd store everything there, using `loadAddonResults()`.

Reply:
Well we need the the search api for this so we can pass multiple guids..

I didn't see the addons within the search results with this info which we need for the AMIntallButton..